kaolin/ops/conversions/pointcloud.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+__all__ = ['pointclouds_to_voxelgrids']
+
+def _points_to_voxelgrids(points, resolution, return_sparse=False):
```

Comment:
maybe add the prefix `_base` like for: https://github.com/NVIDIAGameWorks/kaolin/blob/master/kaolin/ops/mesh/trianglemesh.py#L26

Also maybe we can add some details about the behavior? (i.e: For what I understand, it only convert points from 0 to 1)